### ISSUE TITLE
Default Nornir settings have different structure than the ones used in Plugin Config

### ISSUE THREAD

**Issue by chadell** [nautobot/nautobot-plugin-nornir#47]
### Environment
* nautobot-plugin-nornir version:  

### Expected Behavior

The config used by `NORNIR_SETTINGS` is consistent either is coming from `settings.PLUGINS_CONFIG` or via default `_NORNIR_SETTINGS`, but they look to be different.

### Observed Behavior

default settings: https://github.com/nautobot/nautobot-plugin-nornir/blob/develop/nautobot_plugin_nornir/constants.py#L5-L9
```
_NORNIR_SETTINGS = {
    "inventory": "nautobot_plugin_nornir.plugins.inventory.nautobot_orm.NautobotORMInventory",
    "credentials": "nautobot_plugin_nornir.plugins.credentials.env_vars.CredentialsEnvVars",
    "nornir.core": {"num_workers": 20},
}
```

Definitive settings are taken from Plugin config "nornir_settings" ( if available: https://github.com/nautobot/nautobot-plugin-nornir/blob/develop/nautobot_plugin_nornir/constants.py#L12
and the data is:
```
PLUGINS_CONFIG = {
    "nautobot_plugin_nornir": {
        "nornir_settings": {
            "credentials": "nautobot_plugin_nornir.plugins.credentials.env_vars.CredentialsEnvVars",
            "runner": {
                "plugin": "threaded",
                "options": {
                    "num_workers": 20,
                },
            },
        },
    },
}
```
so, comparing both:
```
"credentials": "nautobot_plugin_nornir.plugins.credentials.env_vars.CredentialsEnvVars",
"runner": {
   "plugin": "threaded",
       "options": {
            "num_workers": 20,
       },
   },
```

So, I see a difference between the defaults, using "nornir.core" to define the workers, and the plugin config using "runner"/"options".

For instance, in one example of library usage, in https://github.com/nautobot/nautobot-plugin-golden-config/blob/0c3e902582d1ce68094d731aa614f35fc28e701f/nautobot_golden_config/nornir_plays/config_backup.py#L114 , it is looking for "runner" that is not defined by default in the runner

Maybe I'm missing something?

**Comment by chadell** [comment]
Also, something that maybe we could review is that, the way it is done today, if we have some custom config, all 3 keys of the default are overwritten ("runner", "credentials" and "inventory"). Maybe we could do a MERGE instead, so only the keys that are present are overwritten, and the others are kept. WDYT?

**Comment by itdependsnetworks** [comment]
Fixed in #48
